rpc/cloudagent/network/networksecuritygroup/moc_cloudagent_networksecuritygroup.proto

```diff
@@ -46,12 +46,13 @@ message NetworkSecurityGroupRule {
 }
 
 message NetworkSecurityGroup {
+	reserved 5; // deprecated group name (now parented to location)
```

Comment:
I am not sure if we need this. Since we did not create NSG in previous version

Reply:
I don't necessarily know if we did, but considering it was released it's possible that someone could have created, so I think the stakes of keeping it reserved are low. If you want to remove the reservation we can, but I'd want to err on the side of being cautious